plugins/CoreHome/angularjs/report-export/reportexport.popover.html

```diff
@@ -0,0 +1,29 @@
+<div class="report-export-popover row">
+
+    <h1>{{ 'General_Export'|translate }} <span ng-bind="reportTitle"></span></h1>
+
+    <div class="col l6">
+        <div piwik-field uicontrol="select" name="format"
+             title="Export format"
+             value="CSV"
+             ng-model="$parent.reportFormat"
+             full-width="true"
+             options="availableReportFormats">
```

Comment:
When opening a layer the selection list does not extend out of bounds and instead extends the dialog contents vertically with scrollbars. However this also happens with the limit control in the visitor segment popovers.

The latter are receiving an `overflow: auto` from `jQuery UI` (`.ui-dialog .ui-dialog-content`) while the modals (as used here) have it set by Materialize (`.modal`). As both these libraries should have their reasons for setting this it might not be nice to overwrite that setting.

Are there any other options like a radio selection to get around this easily?